certbot/cli.py

```diff
@@ -1139,6 +1144,12 @@ def prepare_and_parse_args(plugins, args, detect_defaults=False):  # pylint: dis
     helpful.add(
         "security", "--rsa-key-size", type=int, metavar="N",
         default=flag_default("rsa_key_size"), help=config_help("rsa_key_size"))
+    helpful.add(
+        "security", "--ecdsa-key-size", type=int, metavar="N",
+        default=flag_default("ecdsa_key_size"), help=config_help("ecdsa_key_size"))
+    helpful.add(
+        "security", "--key-type", type=str, choices=['rsa', 'ecdsa'], nargs="+",
```

Comment:
On top of the unit tests added here, I would like an integration test in `certbot-ci/certbot_integration_tests/certbot_tests/test_main.py` to check that certbot behaves as expected when the new flags are added. Typical scenarios I have in mind:
* pass no flag or `--key-type=rsa` and expect a RSA key/cert generated with the default RSA size
* pass `--key-type=ecdsa` and expect an ECDSA key/cert generated with the default ECDSA size
* pass wrong `--key-type` and expect certbot to crash with the expected error message
* handle key size control flags (ECDA and RSA) may be added in a modified version of the existing test `test_renew_with_changed_private_key_complexity`